### PR TITLE
Add todo_include_todo into generated quickstart config

### PR DESCRIPTION
**Pull request by IanLee1521**
Create a `todo_include_todo` entry in the quickstart generated config. Documentation describing this option is at: http://sphinx-doc.org/ext/todo.html

### CODE REVIEW

File: sphinx/quickstart.py [hunk 1]
```diff
@@ -53,6 +53,7 @@
     'epub': False,
     'ext_autodoc': False,
     'ext_doctest': False,
+    'ext_todo': False,
```
Comment (birkenfeld) [inline sphinx/quickstart.py:56]: If you add todo here, you should also add a question to the ask_user function.

File: sphinx/quickstart.py [hunk 2]
```diff
@@ -1284,6 +1288,8 @@ def generate(d, overwrite=True, silent=False):
         d['extensions'] = '\n' + indent + extensions + ',\n'
     else:
         d['extensions'] = extensions
+    print(d['extensions'])
```
Comment (birkenfeld) [inline sphinx/quickstart.py]: These two lines look wrong.

File: test-reqs.txt [hunk 3]
```diff
@@ -1,3 +1,5 @@
+nose
```
Comment (birkenfeld) [inline test-reqs.txt:1]: True but unrelated.

Comment (IanLee1521) [inline test-reqs.txt:1]: Am I misunderstanding that `test-reqs.txt` is the requirements file for testing?

File: sphinx/quickstart.py (hunk 2 above)
Comment (IanLee1521) [inline sphinx/quickstart.py]: Definitely agree with the print, that was an oversight of mine. What is wrong with the `d['ext_todo'] = text_type(d['ext_todo'])` line?

File: test-reqs.txt (hunk 3 above)
Comment (birkenfeld) [inline test-reqs.txt:1]: No, that's correct. Just that this change doesn't relate to the pull request's title. But it's a good change nevertheless.

File: sphinx/quickstart.py (hunk 2 above)
Comment (birkenfeld) [inline sphinx/quickstart.py]: It's not used for the other `ext_` values, so why is it necessary for `ext_todo`? (The replacement for `todo_include_todos` works regardless.)

File: sphinx/quickstart.py (hunk 1 above)
Comment (IanLee1521) [inline sphinx/quickstart.py:56]: Isn't that already asked with: https://github.com/sphinx-doc/sphinx/pull/1712/files#diff-639ce3a52583644f64e3dce953a93a39L1225 ?

File: sphinx/quickstart.py (hunk 2 above)
Comment (IanLee1521) [inline sphinx/quickstart.py]: Ah, good point.

File: sphinx/quickstart.py (hunk 1 above)
Comment (birkenfeld) [inline sphinx/quickstart.py:56]: D'oh. I thought it wasn't among the questions yet. Sorry.

File: test-reqs.txt (hunk 3 above)
Comment (IanLee1521) [inline test-reqs.txt:1]: Ah. Yeah, I think I originally meant to commit it as a separate PR.

File: sphinx/quickstart.py (hunk 1 above)
Comment (IanLee1521) [inline sphinx/quickstart.py:56]: No worries.

**Comment by IanLee1521** [comment]
Looks like the build is failing since https://travis-ci.org/sphinx-doc/sphinx/builds/49779847 ?

**Comment by IanLee1521** [comment]
Any feedback that I missed with this PR?

**Comment by birkenfeld** [comment]
No, I just didn't get around to doing another review. The build is not failing, it's having some network trouble while installing dependencies from PyPI. I don't know how to fix this on our end...

**Comment by birkenfeld** [comment]
Hmm, see travis-ci/travis-ci#2389 - I'll try the workaround presented there.

**Comment by birkenfeld** [comment]
Thanks for the PR!

**Pull request merged**